src/Storage/Storage.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
```

Comment:
We might should not check in this file. 
However, you can contact PSH team to see if the update is acceptable for them.

Reply:
> 

any reason this was added? Don't see AssemblyLoading.csproj got referenced anywhere else.

Reply:
If we don't add AssemblyLoading.csproj project, build Storage.sln from a clean enviroment in Visual studio will fail as following picture:
So currently we need to manually add it when meet the failure.

![image](https://user-images.githubusercontent.com/6926193/227486778-e5ebdd00-f621-44aa-816d-e3192f8a8efb.png)